### ISSUE TITLE
"Add Data" on a specific DataONE Dataset not working

### ISSUE THREAD

**Issue by amoeba** [whole-tale/girder_wholetale#44]
This is something I'll look into, but this dataset doesn't register nicely: https://search.dataone.org/#view/urn:uuid:3249ada0-afe3-4dd6-875e-0f7928a4c171

**Comment by amoeba** [comment]
Ah, 

```
In [3]: D1_lookup('urn:uuid:3249ada0-afe3-4dd6-875e-0f7928a4c171')
---------------------------------------------------------------------------
RestException                             Traceback (most recent call last)
<ipython-input-3-fb8433ea69cc> in <module>()
----> 1 D1_lookup('urn:uuid:3249ada0-afe3-4dd6-875e-0f7928a4c171')

~/src/girder_wholetale/server/dataone_register.py in D1_lookup(path)
    180     logger.debug('Parsed initial PID of {}.'.format(initial_pid))
    181
--> 182     package_pid = find_package_pid(initial_pid)
    183     logger.debug('Found package PID of {}.'.format(package_pid))
    184

~/src/girder_wholetale/server/dataone_register.py in find_package_pid(pid)
     94     # usually leaves us with one.
     95     raise RestException(
---> 96         "Multiple resource maps were found and this is not implemented.")
     97
     98

RestException: Multiple resource maps were found and this is not implemented.
```

**Comment by ThomasThelen** [comment]
Maybe we can link this to #39

**Comment by amoeba** [comment]
Oops, poor form not seeing the existing Issue. Linked both to PR.

**Comment by ThomasThelen** [comment]
Closed in commit 31ab1eee336923d8688aeb7e38fe14e0ca1ec5ed
